### PR TITLE
[ALLUXIO-1721] Translate the page "Accessing-Alluxio-from-Zeppelin.md" into Chinese

### PR DESCRIPTION
**Pull request by saltylin**
https://tachyon.atlassian.net/browse/ALLUXIO-1721
Also fix the links in the english page

### CODE REVIEW

**Comment by ifcharming** [comment]
LGTM

Thank you for the improvement and translation!

File: docs/cn/Accessing-Alluxio-from-Zeppelin.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: global
+title: 通过Zeppelin访问Alluxio
+nickname: 通过Zeppelin访问Alluxio
+group: Under Store
+priority: 3
+---
+
+[Apache Zeppelin](http://zeppelin-project.org/) 是一个能够进行交互式数据分析的网络记事本。早期版本的Alluxio(Tachyon 0.8.2)已经被
```
Comment (RongGu) [inline docs/cn/Accessing-Alluxio-from-Zeppelin.md]: “网络记事本”-->“基于Web的记事本”

File: docs/cn/Accessing-Alluxio-from-Zeppelin.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+layout: global
+title: 通过Zeppelin访问Alluxio
+nickname: 通过Zeppelin访问Alluxio
+group: Under Store
+priority: 3
+---
+
+[Apache Zeppelin](http://zeppelin-project.org/) 是一个能够进行交互式数据分析的网络记事本。早期版本的Alluxio(Tachyon 0.8.2)已经被
+[集成进了Zeppelin](https://github.com/apache/incubator-zeppelin/blob/master/docs/interpreter/tachyon.md)  ，我们将会使用最新版的Alluxio更新该集成功能。
```
Comment (RongGu) [inline docs/cn/Accessing-Alluxio-from-Zeppelin.md]: “早期版本的Alluxio(Tachyon 0.8.2)已经被集成进了Zeppelin”-->"早期版本的Alluxio(Tachyon 0.8.2)已经与Zeppelin进行了集成。"

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**